sdk/servicebus/azservicebus/changelog.md

```diff
@@ -0,0 +1,88 @@
+# Change Log
```

Comment:
The changelog should be updated to start from v0.1.0

Reply:
Changelog updated